docs/2_4_common_schema.adoc

```diff
@@ -1687,7 +1687,7 @@ Elements <<Output>>, <<ContinuousStateDerivative>>, <<ClockedState>>, <<InitialU
 |
 [[dependencies,`dependencies`]]
 Optional attribute defining the algebraic dependencies as list of value references of the unknown latexmath:[{\mathbf{v}_{\mathit{unknown}}}] directly with respect to latexmath:[{\mathbf{v}_{\mathit{known}}}].
-For real valued unknowns and knowns this means that the partial derivative on the unknown with respect to the known is not identical to zero.
+For real valued unknowns and knowns this means that the partial derivative on the unknown with respect to the known is not identically zero.
```

Comment:
For a real valued unknown and and a real valued known this means that the partial derivative of the unknown with respect to the known is not identically zero.

Reply:
"on the unknown" --> of the unknown

I think for a given simulation we cannot guarantee that such a partial derivative is not constant zero; the presence of unknows and knowns in the above ist only means that _structurally_ it could be nonzero.

Reply:
@chrbertsch That is true, what we should say is that not listing a known implies that is has a partial derivative with respect to this known that is identically zero.

I will try to reformulate

Reply:
Does an FMU have to expose all partial derivatives if it sets the corresponding capability flag? Or can it be that it just exposes some, and the others are not exposed but could be nonzero?

Reply:
If an FMU supports getDirectionalDirectional deriavtives, it must be able to compute them for all continuous outputs and all continuousStateDerivatives if they are exposed. An ME FMU must expose all continuousStateDerivatives, while a CS FMU is allowed to hide them. All outputs must be listed in ModelStructure and then their partial derivatives of the continuous real ones should be able to be computed if partial derivatives is supported.
It change an output to a local variable then there is no guarantee that partial derivative functions will work etc.